development-updates.md

```diff
@@ -84,7 +84,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
 | [Ayush](https://github.com/ayushm2003/)                        |                                                                                                                                      |                                                                                                               |                                                                                                                                                                           |
 | [Balaji Pachai](https://github.com/balajipachai/)              |                                                                                                                                      |                                                                                                               |                                                                                                                                                                           |
 | [Behkod](https://github.com/behkod)                            |                                                                                                                                      |                                                                                                               |                                                                                                                                                                           |
-| [Bharath](https://github.com/bharath-123)                      |                                                                                                                                      | [Update 4](https://hackmd.io/1h7enjF6RMe19xQliT3iOA)                                                          |                                                                                                                                                                           |
+| [Bharath](https://github.com/bharath-123)                      |                                                                                                                                      | [Update 4](https://hackmd.io/1h7enjF6RMe19xQliT3iOA)                                                          | [PEPC-Boost](https://hackmd.io/wmVhI999T1eUkuFRgBUz4Q)                                                                                                                                                                           |
```

Comment:
Project proposal column should point to your project doc in `/projects`

Reply:
Done!